docs/_source/_common/tabs/argilla_connect.md

```diff
@@ -40,4 +40,23 @@ client = Argilla(
 ```
 :::
 
-::::
\ No newline at end of file
+:::{tab-item} CLI
+
+From `1.16.0` version you can use the CLI to connect to an Argilla server.
+
+First login to the Argilla server using the CLI
+
+```sh
+argilla login --api-url http://localhost:6900
+```
+
+Then call `init` function without arguments to use the stored credentials created by the `login` command
```

Comment:
FYI does this mean we should always call the `rg.init`? Also with the correct env vars?

Reply:
Indeed there's no need to call `rg.init` explicitly if you want to connect to an Argilla Server that you've previously logged in into 👍🏻 Also this is useful as most of the commands depend on `argilla login` as an entry point, since those require a connection to Argilla, so internally we're initialising the `Argilla` client re-using the credentials dumped in `~/.argilla/credentials.json`